base/tools/src/main/java/com/netscape/cmstools/nss/NSSKeyExportCLI.java

```diff
@@ -47,6 +47,9 @@ public void createOptions() {
         option = new Option(null, "wrapper", true, "Nickname of the wrapper certificate");
         option.setArgName("nickname");
         options.addOption(option);
+
```

Comment:
Is this a new option for some cli?  I don't see any usage update in the code.

Reply:
It's been so long.... I think I was just adding the OAEP flag to this CLI. I don't remember if it's being used . I"ll dig back into it.

Reply:
Looks like no one was calling printHelp which prints out cmd line info along with all the options. This will be addressed.